src/dev_server.rs

```diff
@@ -9,26 +9,77 @@ use std::{
     process,
 };
 
+/// A simple HTTP server that can, optionally, watch a given command.
+///
+/// Get the files at `watch_path` and serve them at a given IP address
+/// (127.0.0.1:8000 by default). An optional command can be run along the
+/// server, relaunched if changes are detected in your project via a [`Watch`].
+///
+/// # Usage
+///
+/// ```rust,no_run
+/// # use std::process;
+/// # use xtask_wasm::{anyhow::Result, clap};
+/// #
```

Comment:
You need to keep those imports visible so the user know where they can find them.

```suggestion
/// use std::process;
/// use xtask_wasm::{anyhow::Result, clap};
///
```